01/Fallen70/wordvalue.py

```diff
@@ -0,0 +1,30 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    return [
+        word.strip()
+        for word in open(DICTIONARY).readlines()
+    ]
+
+def calc_word_value( word ):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    return sum(
+        LETTER_SCORES[letter.upper()]
+        for letter in word
+        if letter.isalpha()
```

Comment:
are all letters in `LETTER_SCORES` _alpha_? consider using `LETTER_SCORES.get` to prevent unexpected `KeyError`s here